src/applications/edu-benefits/0993/helpers.js

```diff
@@ -0,0 +1,11 @@
+import _ from 'lodash/fp';
+
+export function prefillTransformer(pages, formData, metadata) {
+  const newFormData = _.set('verified', !!metadata.verified, formData);
```

Comment:
The form has a different presentation based on whether the applicant is verified or not, we store that information in the form data here so it can be used by `hideIf`, which uses form data as a parameter, similar to how this is handled by vic-v2. @lihanli @rianfowler is the FE the appropriate place to set this?